lib/reporters.js

```diff
@@ -23,9 +23,10 @@ NullReporter.prototype.report = function(series, onSuccess) {
 // DataDogReporter
 //
 
-function DataDogReporter(apiKey, appKey) {
+function DataDogReporter(apiKey, appKey, apiHost) {
     apiKey = apiKey || process.env.DATADOG_API_KEY;
     appKey = appKey || process.env.DATADOG_APP_KEY;
+    this.apiHost = apiHost || process.env.DATADOG_API_HOST;
```

Comment:
I made this public so it can be easily inspected, and it’s not sensitive like the API key and app key are. LMK if that’s not OK.